pkg/jaeger/query/query.go

```diff
@@ -41,47 +45,85 @@ func (p *Query) SpanWriter() spanstore.Writer {
 }
 
 func (p *Query) GetTrace(ctx context.Context, traceID model.TraceID) (*model.Trace, error) {
+	metrics.RequestsTotal.With(prometheus.Labels{"type": "trace", "handler": "get_trace", "code": ""}).Inc()
```

Comment:
What is the case in which we do have a request, but it doesn't have a response code?